frontend/app/components/admin-component.js

```diff
@@ -0,0 +1,7 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  willDestroyElement() {
```

Comment:
This will get called when it is destroying the component, but there is no logic here as it is initalising a local constructor for the component and that's it? Use case here?